tests/test-team/template-mgmt-component-tests/template-mgmt-manage-templates-page.component.ts

```diff
@@ -0,0 +1,298 @@
+import { test, expect } from '@playwright/test';
+import { v4 as uuid } from 'uuid';
```

Comment:
Any reason you can't use `node:crypto` instead? 

```
import { randomUUID } from 'node:crypto'
```

Reply:
I wasn't aware of its existence. As we are very unlikely to use this code outside of node.js, I'm happy to switch over.

Reply:
Change made.